tests/utils.c

```diff
@@ -6,9 +6,57 @@
 #include <glib.h>
 #include <glib/gstdio.h>
 #include <gio/gio.h>
+#include <parted/parted.h>
+#include "pu-glib-compat.h"
 #include "pu-utils.h"
 #include "pu-error.h"
 
+static GFile *
+create_tmp_file(const gchar *filename,
+                const gchar *pwd,
+                gsize size,
+                GError **error)
+{
+    GFile *file;
+    g_autoptr(GFileIOStream) stream = NULL;
+    g_autofree gchar *buffer = NULL;
+
+    file = g_file_new_build_filename(pwd, filename, NULL);
+    g_assert_nonnull(file);
+    buffer = g_new0(gchar, size);
+    g_assert_true(g_file_replace_contents(file, buffer, size, NULL, FALSE, 0, NULL, NULL, error));
+    g_assert_no_error(*error);
+
+    return g_steal_pointer(&file);
+}
+
+typedef struct {
+    GError *error;
+    GFile *part;
+    gchar *path;
+} EmptyFileFixture;
+
+static void
+empty_file_set_up(EmptyFileFixture *fixture,
+                 G_GNUC_UNUSED gconstpointer unused)
```

Comment:
Rename the variable `unused` to `user_data`, to be in line with `GTestFixtureFunc`. Using `G_GNUC_UNUSED` is enough indication that this variable remains unused.